packages/playground/src/utils/filter_farms.ts

```diff
@@ -0,0 +1,78 @@
+import type { CertificationType, Farm, FarmsQuery, Pagination } from "@threefold/gridproxy_client";
```

Comment:
Instead of recreating this component, we can definitely utilize the `node-filter` component. If there are any obstacles preventing its use, we can modify it to make it more versatile and reusable.